### ISSUE TITLE
Zigbee2MQTT nodes not working properly since 9.1.2

### ISSUE THREAD

**Issue by datbilling** [hassio-addons/addon-node-red#1039]
# Problem/Motivation

After upgrading from 9.1.1 to 9.1.2 or 9.1.3, my Zigbee2MQTT nodes aren’t working properly. 

## Expected behavior

That the zigbee devices from z2m still output their state when the events: state node is triggered.

## Actual behavior

When debugging a node such as an aqara wireless switch, there is still output when I click the switch button, but it is ‘undefined’ rather than the action such as ‘single’ or ‘double’. So the flow stops right there. Same case for my door contact and motion sensor. Possibly for all devices added to Z2M, but haven’t tested others.

## Steps to reproduce

Upgrade to 9.1.2 or 9.1.3. Check the debug messages for a Zigbee2MQTT sensor such as an aqara wireless switch Events: state node after pressing the switch to trigger a state changed event. See that the output is now ‘undefined’, so the actual state isn’t output to the next attached node. 

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by frenck** [comment]
Do they end up in MQTT?

**Comment by datbilling** [comment]
> Do they end up in MQTT?

Do you mean are the states shown in Zigbee2MQTT?

The state is shown in the Z2M log after I click the aqara button, for example here it is 'single':
_Info MQTT publish: topic 'zigbee2mqtt/Aqara wireless switch (with gyroscope) 2', payload '{"action":"single","battery":100,"click":"single","linkquality":123,"voltage":3025}'
Info MQTT publish: topic 'zigbee2mqtt/Aqara wireless switch (with gyroscope) 2', payload '{"action":"","battery":100,"linkquality":123,"voltage":3025}'
Info MQTT publish: topic 'zigbee2mqtt/Aqara wireless switch (with gyroscope) 2', payload '{"battery":100,"click":"","linkquality":123,"voltage":3025}'
Info MQTT publish: topic 'zigbee2mqtt/Aqara wireless switch (with gyroscope) 2/action', payload 'single'
Info MQTT publish: topic 'zigbee2mqtt/Aqara wireless switch (with gyroscope) 2/click', payload 'single'
Info MQTT publish: topic 'zigbee2mqtt/Moes Thermostatic Radiator Valve Kitchen', payload_

Whereas when debugging the node in node-red, I get this:
_msg.payload : undefined
undefined
msg.payload : undefined
undefined_

I just tried updating node-red to 9.1.4 but it has the same issue.

**Comment by datbilling** [comment]
I just tried creating a new 'events: state' node for the switch and it works correctly in 9.1.4. I found the difference was that the new node has these output properties:
![image](https://user-images.githubusercontent.com/58982799/121823364-16419900-cca5-11eb-9776-20d3d0468ee3.png)

My existing nodes that don't work have no output properties. 

Where these ouput properties made compulsory in 9.1.2?
Is there a quick way to bulk edit all the exisitng nodes that have the missing output properties?

**Comment by datbilling** [comment]
After restoring some old flows.json exports, I found on May 21st most, possibly all my 'state nodes' and 'current state nodes' lost their output properties. I don't know what caused that to happen. 

It took a long time but I've now manually added the output properties back to the many nodes. They are now working on node-red version 9.1.4

I also don't know why it was only from version 9.1.2 that the missing output properties were a problem.
